test/features/email/domain/usecases/calendar_event_accept_interactor_test.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:dartz/dartz.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:jmap_dart_client/jmap/account_id.dart';
+import 'package:jmap_dart_client/jmap/core/id.dart';
+import 'package:jmap_dart_client/jmap/mail/calendar/properties/event_id.dart';
+import 'package:jmap_dart_client/jmap/mail/calendar/reply/calendar_event_accept_response.dart';
+import 'package:mockito/annotations.dart';
+import 'package:mockito/mockito.dart';
+import 'package:tmail_ui_user/features/email/domain/exceptions/calendar_event_exceptions.dart';
+import 'package:tmail_ui_user/features/email/domain/repository/calendar_event_repository.dart';
+import 'package:tmail_ui_user/features/email/domain/state/calendar_event_accept_state.dart';
+import 'package:tmail_ui_user/features/email/domain/usecases/calendar_event_accept_interactor.dart';
+
+import 'calendar_event_accept_interactor_test.mocks.dart';
+
+@GenerateNiceMocks([MockSpec<CalendarEventRepository>()])
+void main() {
+  final calendarEventRepository = MockCalendarEventRepository();
+  final acceptCalendarEventInteractor = AcceptCalendarEventInteractor(calendarEventRepository);
+  final accountId = AccountId(Id('123'));
+  final blobId = Id('123321');
+
+  group('calendar event accept interactor should emit expected states', () {
```

Comment:
Should only describe the object to be inspected. We write more details in each `test` case

Reply:
Done in:
- https://github.com/linagora/tmail-flutter/pull/2803